src/pineko/evolve.py

```diff
@@ -1,6 +1,7 @@
 """Tools related to evolution/eko."""
 
 import copy
+import json
```

Comment:
Is this really needed?

Reply:
after #166 I would say yes, do you agree?